application/src/Mvc/Controller/Plugin/UserIsAllowed.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Omeka\Mvc\Controller\Plugin;
 
+use Omeka\Api\ResourceInterface;
```

Comment:
Same objection to the `use`s in this commit.

Reply:
Ok. Fixed in next commit.